### PR TITLE
Play 3.0.x and Pekko updates

### PR DESCRIPTION
**Pull request by sentenza**
In order to use `op-rabbit-play-json` with Play v3.0.x this library should target Play 3, which is the only version supporting Pekko.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -19,13 +19,13 @@ val commonSettings = Seq(
...
   libraryDependencies ++= Seq(
-    "com.chuusai" %%  "shapeless" % "2.3.9",
-    "com.typesafe" % "config" % "1.4.2",
+    "com.chuusai" %%  "shapeless" % "2.3.10",
+    "com.typesafe" % "config" % "1.4.3",
     "com.github.pjfanning" %% "pekko-rabbitmq" % "7.0.0",
-    "com.rabbitmq" % "amqp-client" % "5.18.0",
-    "org.slf4j" % "slf4j-api" % "1.7.36",
+    "com.rabbitmq" % "amqp-client" % "5.20.0",
+    "org.slf4j" % "slf4j-api" % "2.0.9",
```
Comment (pjfanning) [inline build.sbt:26]: slf4j 2 upgrade is a bit of a mess - pekko itself uses slf4j 1 - and there are compatibility issues between the 2 versions

File: build.sbt [hunk 2]
```diff
@@ -19,13 +19,13 @@ val commonSettings = Seq(
...
     "com.spingo" %% "scoped-fixtures" % "2.0.0" % Test,
-    "ch.qos.logback" % "logback-classic" % "1.2.12" % Test,
+    "ch.qos.logback" % "logback-classic" % "1.4.14" % Test,
```
Comment (pjfanning) [inline build.sbt:28]: 1.4.14 only supports java 11 - I will not take this change

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.9.6
+sbt.version=1.9.8
```
[the remark this answers is not in the document]
Comment (sentenza) [reply project/build.properties:1]: What about SBT 1.9.7?

[the remark this answers is not in the document]
Comment (pjfanning) [reply project/build.properties:1]: I have committed a few changes, including 1.9.7 - just to break this PR up a bit

File: build.sbt (hunk 1 above)
Comment (sentenza) [reply build.sbt]: Reverted

**Comment by pjfanning** [comment]
seems fine - I'll set up a v4.0.0 release - probably best to treat it is a major change due to the group id change on play libs

**Comment by sentenza** [comment]
@pjfanning thank you. Yes, this is not retro-compatible with Play 2.9 and in general Pekko can only be used with Play 3, so the previous version of this library cannot really be used either with Play 2.9 or Play 3. 

I believe that those two issues are also related to wrong transitive dependencies imported via `op-rabbit:3.0.0`: https://github.com/lloydmeta/enumeratum/issues/388 and https://github.com/playframework/play-grpc/issues/499.

**Comment by pjfanning** [comment]
op-rabbit 4.0.0-SNAPSHOT is published with these changes

**Comment by sentenza** [comment]
@pjfanning I've already tested v4.0.0 and it's working as expected, solving all the cross-dependency issues. Thank you. :)

**Pull request merged**